docs: update sql-editor screenshots (#736)
bytebase/bytebase.com: content/docs/sql-editor/admin-mode.md

## Run query in Admin Mode

> 

Reviewer: Can you change Line 7 to 

"Please note that you should be really careful when using Admin Mode, since you are taking full control of your database."

Author: updated.

> Click **Admin Mode** button to create an Admin Mode tab. Note that this is only available for DBAs and Workspace Owners.

Reviewer: ```suggestion
Click **Admin Mode** button to open a separate Admin Mode tab. Note that this is only available for DBAs and Workspace Owners.
```
